### DISCUSSION TITLE
Remove Empty Pages

### DISCUSSION THREAD

**Discussion by Lownex** [jonaswinkler/paperless-ng#1702] [Feature Requests]
Ladies and Gents,

I am not sure if this idea may have been already evaluated. My suggestion for a new feature would be the removal of obviously empty pages on a scan. Most of the private users will not have Pro-Solutions in terms of scanning documents. Even if, some solutions are not very powerful when it comes to detect empty pages in scanned documents. Even if scanners claim to have detection capabilities for empty pages quite often it's quite poor.

My Suggestion e.g.: If the amount of printing is below ~3% and there is no OCR-readable detection, it will be most likely an empty page. It happens quite often, that paper is very thin or there was some stuff from a perforator or simple folding on the document.

Regards
Lownex

**Comment by kenwiesner** [comment]
I've evaluated a few solutions and some of them had this built in. This would be a great feature for Paperless-ng. More often than not I'll have docs that have a mix of single side and double sided documents. Having this feature or at the very least a way when viewing the doc to manually remove a blank page would be great!

**Reply by Lownex** [reply]
Hey Ken, it seems that Jonas is currently not working on that project, check the fork paperless-ngx and try to push that idea there.

cheers
